flutter_local_notifications/lib/src/platform_flutter_local_notifications.dart

```diff
@@ -168,6 +169,13 @@ class AndroidFlutterLocalNotificationsPlugin
         return Future.error('method not defined');
     }
   }
+
+  /// Creates a notification channel.
+  ///  
+  /// Only applies to Android 8.0+
+  Future<bool> createNotificationChannel(AndroidNotificationChannel notificationDetails) async {
```

Comment:
See comment made previously on making this return `Future<void>` as the Android code never returns false